install-deps

```diff
@@ -110,6 +110,12 @@ elif [[ "$(uname)" == 'Linux' ]]; then
     # Detect archlinux
     elif [[ "$DISTRO" = "arch" ]]; then
         distribution="archlinux"
+    # Detect deepin
+    elif [[ "$DISTRO" = "deepin" ]]; then
+        export DEBIAN_FRONTEND=noninteractive
+        distribution="deepin"
+        deepin_major_version="${VERSION%%.*}"
+    
     # Detect Ubuntu
     elif [[ "$DISTRO" = "ubuntu" ]]; then
```

Comment:
isn't deepin an ubuntu derivative? in that case instead of replicating code, cant you just add it here